webapp/channels/src/packages/mattermost-redux/src/utils/post_list.ts

```diff
@@ -58,7 +58,7 @@ export function makeFilterPostsAndAddSeparators() {
         (state: GlobalState, {postIds}: PostFilterOptions) => getPostsForIds(state, postIds),
         (state: GlobalState, {lastViewedAt}: PostFilterOptions) => lastViewedAt,
         (state: GlobalState, {indicateNewMessages}: PostFilterOptions) => indicateNewMessages,
-        (state) => state.entities.posts.selectedPostId,
+        () => '', // This previously returned state.entities.posts.selectedPostId which stopped being set at some point
```

Comment:
Shouldn't we remove this completely? Or is for some kind of backwards compatibility?

Reply:
I would've removed it, but I'm suspicious that there's something potentially broken here. I don't remember what exactly this was for since I remember it always being weird, so I can't confirm if there's actually a bug, but I wanted to leave a comment to remind someone of it if they're looking at this in the future. In other words, this code has something weird happening, so I wanted a weird comment to fit that 😛

I did find that this code got copied into `selectors/views/threads` for some reason and slightly modified which removed this part. That seems to be around the same time that `state.entities.posts.selectedPostId` got removed, but I didn't have time to unravel all of that right now